Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/check_mark_button.tsx

```diff
@@ -9,27 +9,26 @@ interface Properties {
   /** Determines if the check mark is readonly or not. */
   isReadOnly: boolean;
```

Comment:
Is this prop needed anymore? Does the button look or behave differently if it's read only?

Reply:
It does. If's it's readonly the button should never have it's color change on click.

Reply:
This prop isn't used anywhere though. Try removing the prop and the single reference to it (which is in an ```if``` statement). Basically the idea is that any button you ever design can be read-only if the user of that button ignores the onclick. The only exceptions to this will be certain buttons that have a different style or effect depending on whether it's read only, but for this Component there is no style or effect that depends on that prop.